src/component.js

```diff
@@ -13,9 +13,19 @@ export default class Component {
     constructor(el, seedDataForCloning = null) {
         this.$el = el
 
-        const dataAttr = this.$el.getAttribute('x-data')
+        let dataAttr = this.$el.getAttribute('x-data')
+
+        if (! dataAttr && this.$el.getAttribute('data-x-data')) {
+            dataAttr = this.$el.getAttribute('data-x-data')
+        }
```

Comment:
We can also use a one liner
`const dataAttr = this.$el.getAttribute('x-data') || this.$el.getAttribute('data-x-data')`

Reply:
I did use OR syntax originally but I went for the more verbose and clear equivalent. Happy to change it though if you think it's better :)

Reply:
I agree with changing to `||`s, it'll keep it more compact and the 2 `getAttribute` calls are " the same" (represent the same operation).

Reply:
Yeah, if you don't mind changing it, we'll save a few lines. 👍